City-of-Pori/pori-kada
# PORI-132 toimipisteet sivulla

Add _Related service offering_ field to _page_ content type. The field references service offering nodes and displays the _place_ nodes referenced inside the _service_offer_.

To test:
```
git checkout PORI-132-toimipisteet-sivulla 
drush fra -y ; drush cc all
```
Update a _page_ node by referencing one or more items in the _Related service offering_ field on the content edit form. Save the _page_ and see if the selected items are displayed on the bottom of the page. (Note: the output will be un-styled).
